.github/workflows/build_docker_image.yml

```diff
@@ -57,10 +63,10 @@ jobs:
           TRUSTPILOT_API_KEY: ${{ secrets.TRUSTPILOT_API_KEY }}
 
       - name: Run Build Docker
-        run: docker build -t ${{ secrets.WEB_ACCESS_DOCKERHUB_USERNAME }}/$tag_name . --platform=linux/amd64
+        run: docker build -t ${{ secrets.WEB_ACCESS_DOCKERHUB_USERNAME }}/${{ steps.branch_name.outputs.branch_name }} . --platform=linux/amd64
       
       - name: Run Tag Docker
-        run: docker tag ${{ secrets.WEB_ACCESS_DOCKERHUB_USERNAME }}/$tag_name ${{ secrets.WEB_ACCESS_DOCKERHUB_USERNAME }}/$tag_name
+        run: docker tag ${{ secrets.WEB_ACCESS_DOCKERHUB_USERNAME }}/${{ steps.branch_name.outputs.branch_name }} ${{ secrets.WEB_ACCESS_DOCKERHUB_USERNAME }}/${{ steps.branch_name.outputs.branch_name }}
```

Comment:
[Q] where is the lowercase function being called for branch name here?

Reply:
you can check line no30 - 35, that's the function we are running to convert Capital letter to lower case 😎